src/main/java/com/intuit/graphql/orchestrator/federation/ExternalValidator.java

```diff
@@ -0,0 +1,22 @@
+package com.intuit.graphql.orchestrator.federation;
+
+import com.intuit.graphql.graphQL.Directive;
+import com.intuit.graphql.graphQL.FieldDefinition;
+import com.intuit.graphql.orchestrator.federation.exceptions.ExternalFieldNotFoundInBaseException;
+import com.intuit.graphql.orchestrator.stitching.StitchingException;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.intuit.graphql.orchestrator.utils.XtextTypeUtils.getFieldDefinitions;
+
+public class ExternalValidator {
+    public void validatePostMerge(EntityTypeMerger.EntityMergingContext entityMergingContext, FieldDefinition fieldDefinition, Directive directive) {
```

Comment:
not sure if I missed it, looks like directive is unused?

Reply:
oh in the case it is not; I was practicing on creating a generic interface while doing that...I'll go ahead and remove that